responsibleai/responsibleai/rai_insights/rai_insights.py

```diff
@@ -168,28 +168,76 @@ def __init__(self, model: Optional[Any], train: pd.DataFrame,
         if model is not None:
             # Cache predictions of the model
             self.predict_output = model.predict(
-                test.drop(columns=[target_column]))
+                self.get_test_data(
+                    test_data=test).drop(columns=[target_column]))
             if hasattr(model, SKLearn.PREDICT_PROBA):
                 self.predict_proba_output = model.predict_proba(
-                    test.drop(columns=[target_column]))
+                    self.get_test_data(
+                        test_data=test).drop(columns=[target_column]))
             else:
                 self.predict_proba_output = None
         else:
             self.predict_output = None
             self.predict_proba_output = None
 
+    def get_train_data(self):
+        """Returns the training dataset after dropping
+        features if such a list is configured.
```

Comment:
```suggestion
        """Returns the training dataset after dropping
        features if any were configured to be dropped.
```
Sorry, nitpicking again. There's not argument to this method so "such a list" confused me a bit.

Reply:
I suppose same at the next method.

Reply:
Always like to hear about comments on comments :D